src/field/FieldParamsPointers.hh

```diff
@@ -0,0 +1,79 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file FieldParamsPointers.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "base/Macros.hh"
+#include "base/Types.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Host and device parameters for propagation in a magnetic field and
+ * suggested default values
+ */
+struct FieldParamsPointers
+{
+    //! the minimum length of the field step = 1.0e-5 [mm]
+    real_type minimum_step;
+
+    //! the closest miss distrance = 0.25 [mm]
+    real_type delta_chord;
```

Comment:
Is the value in the doc an arbitrary example or a good default value?  If the later consider using:
```suggestion
    //! the closest miss distrance = 0.25 [mm]
    real_type delta_chord = 0.25;
```

Reply:
Set default values as suggested.